iam_policies.tf

```diff
@@ -295,7 +295,7 @@ resource "aws_iam_role_policy_attachment" "shepherd_redshift" {
   // https://stackoverflow.com/questions/45486041/how-to-attach-multiple-iam-policies-to-iam-roles-using-terraform
   for_each = toset([
     "arn:aws-us-gov:iam::aws:policy/AmazonRedshiftDataFullAccess",
```

Comment:
we can probably drop the DataFullAccess role now.

Reply:
Spoke with @eli-dds about this and he is would like to keep API access until RedShift is fully tested on our end.